chainer/functions/activation/clipped_relu.py

```diff
@@ -155,3 +155,38 @@ def clipped_relu(x, z=20.0):
     """
     y, = ClippedReLU(z).apply((x,))
     return y
+
+
+def relu6(x):
+    """Rectifier Unit function clipped at 6.
+
+    It computes
+
+    .. math:: \\text{ReLU6}(x) = \\min(\\max(0, x), 6).
+
+    Args:
+        x (:class:`~chainer.Variable` or :ref:`ndarray`):
+            Input variable. A :math:`(s_1, s_2, ..., s_n)`-shaped float array.
+
+    Returns:
+        ~chainer.Variable: Output variable. A
+        :math:`(s_1, s_2, ..., s_n)`-shaped float array.
+
+    .. seealso:: :func:`chainer.functions.clipped_relu`
+
+    .. admonition:: Example
+
+        >>> x = np.random.uniform(-100, 100, (10, 20)).astype(np.float32)
```

Comment:
I prefer a more concrete example, e.g.
```
>>> x = np.array([-20, -2, 0, 2, 4, 10, 100], np.float32)
>>> x
(...)
>>> F.relu(x)
Variable(...)
```

@Crissman What do you think?

Reply:
@toslunar Thanks for the review. I have improved the example as you suggested. Is this fine?